src/main/java/pl/net/was/JsonTrinoConverter.java

```diff
@@ -258,6 +262,7 @@ private static void writeTo(BlockBuilder rowBuilder, Object value, Type type)
         }
         if (type instanceof DateType dateType) {
             dateType.writeInt(rowBuilder, (Integer) value);
+            return;
```

Comment:
Put this into a separate commit. I'm trying to add more data types to the fastapi example app to have coverage for this.

Reply:
Fixed in 7c93b149d7130701e991b5613136f2ff48d1f7ac, you can rebase.

Reply:
> Put this into a separate commit. I'm trying to add more data types to the fastapi example app to have coverage for this.

Do you mean the tests? Ok

Reply:
It's not needed anymore